src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -1553,10 +1553,13 @@
             { \\"value\\": [ \\"australiaeast\\", \\"eastus\\", \\"japaneast\\" ] } }"
   - name: Create a resource policy assignment with a system assigned identity.
     text: >
-        az policy assignment create --name myPolicy --policy {PolicyName} --assign-identity
+        az policy assignment create --name myPolicy --policy {PolicyName} --assign-identity [system] --location eastus
```

Comment:
Please refer to guide this doc [managed_identity_command_guideline](https://github.com/Azure/azure-cli/blob/dev/doc/managed_identity_command_guideline.md) to design managed identity related commands and parameters
So we should use `--mi-system-assigned` instead of `--identity [system]`

Reply:
Updated.